src/__tests__/FileCollection.test.ts

```diff
@@ -10,7 +10,7 @@ describe('FileCollection', () => {
     );
 
     const filtered = fileCollection.filter((file) =>
-      file.name.match(/[ef]\.txt/),
+      /[ef]\.txt/.exec(file.name),
```

Comment:
This change is not required by our last formatting rules. You should update the dependencies

Reply:
ok